src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/StrTransForm.cs

```diff
@@ -0,0 +1,288 @@
+﻿using CoreWCF.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Xml;
+
+namespace CoreWCF.IdentityModel
+{
+    class StrTransform : Transform
+    {
```

Comment:
StrTransform is only needed for full message security. It's not supported by the crypto SignedXml either.

Reply:
should we keep it or not ? I think we should support message security in long run..